data/avro-schema/src/main/resources/avro/net/corda/data/crypto/wire/ops/encryption/request/DecryptRpcCommand.avsc

```diff
@@ -0,0 +1,28 @@
+{
+  "type": "record",
+  "name": "DecryptRpcCommand",
+  "namespace": "net.corda.data.crypto.wire.ops.encryption.request",
+  "doc": "Request to decrypt the given byte array",
+  "fields": [
+    {
+      "name": "category",
```

Comment:
nit: so this means we could be passing in any category. So for example `LEDGER` (ledger keys are meant to be used for signing) to decrypt secrets. Should we be restricting this field with an enum of valid categories?